.travis.yml

```diff
@@ -1,6 +1,13 @@
 sudo: required
 dist: trusty
 language: c
+
+# Build only master and stable branches.  Other branches got through PRs.
```

Comment:
got -> go ?

The `3.5` value won't really work since the 3.5 branch does not contain this commit and thus does not forbid itself to be built. I guess it is still a good idea to have this, I just wanted to mention it.